types/hapi/index.d.ts

```diff
@@ -1938,7 +1938,7 @@ export class Request extends Podium {
      * an object containing the query parameters.
      * TODO update with outcome of: https://github.com/hapijs/hapi/pull/3479
      */
-    query: Object;
+    query: any;
```

Comment:
👍 

Would be nice to add a test for this like: `./[hapi/test/request](https://github.com/mailto1587/DefinitelyTyped/tree/master/types/hapi/test/request)/query.ts` with the contents of:

    // Added test in addition to docs, for request.query

    import * as Hapi from 'hapi';

    interface GetThingQuery {
        name: string;
    }

    const handler: Hapi.RouteHandler = function (request, reply) {

        const query = request.query as GetThingQuery;

        return reply(`You asked for ${query.name}`);
    };